privaterelay/settings.py

```diff
@@ -105,6 +105,7 @@
 AWS_SNS_TOPIC = config('AWS_SNS_TOPIC', None)
 AWS_SNS_KEY_CACHE = config('AWS_SNS_KEY_CACHE', 'default')
 AWS_SES_CONFIGSET = config('AWS_SES_CONFIGSET', None)
+AWS_SQS_QUEUE_URL = config('AWS_SQS_QUEUE_URL', None)
```

Comment:
Same for this, since this is only being used in the command should we consider getting this inside the command script?

Reply:
Hmm ... I like having ALL the app configs in a single module - e.g., `settings`. There are a number of other app configs that are only used in a single module but we still instantiate them in `settings`.